unity-client/Assets/Tutorial/Prefabs/Common/TutorialRobot/Materials/White_MTL.mat

```diff
@@ -9,17 +9,21 @@ Material:
   m_PrefabAsset: {fileID: 0}
   m_Name: White_MTL
   m_Shader: {fileID: 4800000, guid: 7bc4dc8666ba3c94e80873a8e72e5b06, type: 3}
-  m_ShaderKeywords: VERTEX_COLOR_ON _EMISSION _NORMALMAP
-  m_LightmapFlags: 1
-  m_EnableInstancingVariants: 1
+  m_ShaderKeywords: 
+  m_LightmapFlags: 6
+  m_EnableInstancingVariants: 0
   m_DoubleSidedGI: 0
-  m_CustomRenderQueue: 2302
```

Comment:
As we are changing the `renderQueue` here, please make sure we aren't breaking anything with this change.